CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Tarmo Pikaro
+# Distributed under the Boost Software License, Version 1.0.
+# See accompanying file LICENSE_1_0.txt or copy at https://www.boost.org/LICENSE_1_0.txt
+
+cmake_minimum_required( VERSION 3.5 )
+project( BoostDateTime LANGUAGES CXX)
+
+add_library( boost_date_time
```

Comment:
As of 1.73 the library is just a stub for backward compatibility -- I don't think we should propagate the idea that it is needed in the cmake?